partner-portal/src/UI/angular.json

```diff
@@ -25,6 +25,7 @@
           "options": {
             "outputPath": "dist",
             "index": "src/index.html",
+            "preserveSymlinks": true,
```

Comment:
for my knowledge, what does this do?

Reply:
This is for the shared library working in the parent project to get the angular references and imports.